src/css/librarybrowser.css

```diff
@@ -624,7 +646,8 @@
     align-items: center;
     -webkit-flex-wrap: wrap;
     flex-wrap: wrap;
-    margin: 1em 0
+    margin: 1em 0;
+    padding-right: 3%;
```

Comment:
Can you use the `padded-left` and `padded-right` classes that already exist rather than add it manually?